mantidproject/mantid
# Remove phase calculations from transform tab

Merge after #25545

**Description of work.**

This updates the transform tab of FDA analysis to use inputs generated by the other tabs. This has led to several options being removed from the transform tab as they are handled elsewhere. In particular:
 * The calculation of the phase table, PhaseQuad and associated options is now done in the phase tab so the options todo with this are gone replaced with an option to select an already calculated PhaseQuaf workspace in the fft case and an option to select an already calculated phase table ion the maxent case.
* The FirstGoodTime and LastGoodTime are now taken from the home tab so no longer appear in this tab.

**To test:**

* Open the frequency domain analysis GUI and check that the outputs from doing FFT and Maxent calculations are the same as they were previously.



Fixes #24446 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
